### DISCUSSION TITLE
Ibiza 2021 Square Model working

### DISCUSSION THREAD

**Discussion by Northfox7** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#360] [General]
First, BIG "Thank You!" for starting and maintaining this project!

I got it to work with an Ibiza 2021 model with square pump. Model number seems 60015.
Has 6 wires and I used the PCB V1 according to the instructions. 
In model.h file it is configured as Miami 2021 without hydrojets. 

Also got it integrated via MQTT with Openhab2 and works with only minor leftovers to be solved.

**Comment by visualapproach** [comment]
You're very welcome! Thanks for your report, I put it in the compatibility table.
